frontend/talentsearch/src/js/components/profile/ProfilePage/ProfilePage.tsx

```diff
@@ -340,7 +920,74 @@ export const ProfilePage: React.FC<ProfilePageProps> = ({
                   })}
                 </Link>
               </div>
-              <p>Diversity and inclusion details</p>
+              <div
+                data-h2-bg-color="b(gray)"
+                data-h2-padding="b(all, m)"
+                data-h2-radius="b(s)"
+              >
+                {(!isWoman || isWoman === null) &&
+                  (!isIndigenous || isIndigenous === null) &&
+                  (!isVisibleMinority || isVisibleMinority === null) &&
+                  (!hasDisability || hasDisability === null) && (
+                    <p>
+                      {intl.formatMessage({
+                        defaultMessage:
+                          "You have not identified as a member of any employment equity groups.",
+                        description:
+                          "Message indicating the user has not been marked as part of an equity group",
+                      })}
+                    </p>
+                  )}
+                {(isWoman ||
+                  isIndigenous ||
+                  isVisibleMinority ||
+                  hasDisability) && (
+                  <div>
+                    <p>
+                      {intl.formatMessage({
+                        defaultMessage: "I identify as:",
+                        description:
+                          "Label preceding what groups the user identifies as part of, followed by a colon",
+                      })}{" "}
+                    </p>{" "}
+                    <ul data-h2-padding="b(left, l)">
+                      {isWoman && (
+                        <li data-h2-font-weight="b(700)">
+                          {intl.formatMessage({
+                            defaultMessage: "Woman",
```

Comment:
I feel like we use these EE groups everywhere even though they're not a graphql enum.  I suggest adding them to the localized constants file.